service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/documents/ManageDocumentsLAController.java

```diff
@@ -173,6 +176,30 @@ public AboutToStartOrSubmitCallbackResponse handleFurtherEvidenceMidEvent(@Reque
         return respond(caseDetails);
     }
 
+    @PostMapping("/final-check/mid-event")
+    public AboutToStartOrSubmitCallbackResponse handleFinalCheckMidEvent(@RequestBody CallbackRequest request) {
+        CaseDetails caseDetails = request.getCaseDetails();
+        CaseData caseData = getCaseData(caseDetails);
+
+        List<String> errors = new ArrayList<>();
+
+        switch (caseData.getManageDocumentLA().getType()) {
```

Comment:
Maybe we could use an if statement here instead of a switch case block as there is only a single case